etc/config/c.amazon.properties

```
@@ -2704,6 +2704,13 @@ compiler.lc3-trunk.name=LC3 (trunk)
 compiler.lc3-trunk.exe=/opt/compiler-explorer/C-LC3-Compiler-trunk/lc3-compile
 compiler.lc3-trunk.isNightly=true
 
+#################################
+# UPMEM DPU Compiler
+group.upmem-clang.compilers=upmem-clang-2023-2
+group.upmem-clang.groupName=UPMEM DPU Clang
+compiler.upmem-clang-2023-2.exe=/opt/compiler-explorer/upmem-2023.2.0-Linux-x86_64/bin/dpu-upmem-dpurte-clang
```

```suggestion
compiler.upmem-clang-2023-2.exe=/opt/compiler-explorer/upmem-dpu/upmem-2023.2.0-Linux-x86_64/bin/dpu-upmem-dpurte-clang
```

Something like that would make things cleaner when we install several toolchain for this proprietary target.